apps/indexer/src/chain/chainSync.ts

```diff
@@ -272,7 +295,7 @@ async function insertBlocks(startHeight: number, endHeight: number) {
 
     blocksToAdd.push(blockEntry);
 
-    if (blocksToAdd.length >= 500 || i === endHeight) {
+    if (blocksToAdd.length >= 50 || i === endHeight) {
```

Comment:
We still want to keep this to 50?

Reply:
Yeah, it keeps the db transaction size more reasonable now that we also inserts the events & attributes.